sbin/txt-tpm1-evt-log-parser.awk

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/gawk -bf
```

Comment:
Why this file has `txt-` prefix but the other one for TPM2 not?

Reply:
https://github.com/TrenchBoot/qubes-antievilmaid/pull/7/commits/2ccd86f8c75b9c905ff792c1b06f3de7bbf96361:

> Contrary to TPM 2.0 format, this one is specific to Intel TXT and
cannot be used for parsing firmware entries, hence the seemingly
inconsistent file name.

Reply:
So tpm2-evt-log-parser.awk can be used for both firmware and TXT?